@xen-orchestra/fs/src/local.js

```diff
@@ -39,6 +40,10 @@ export default class LocalHandler extends RemoteHandlerAbstract {
         })
   }
 
+  async _getInfo() {
```

Comment:
You can remove `async` here, `await` is not used in this function (and you don't need to guarantee the result  is wrapped in a promise, `Abstract` will handle both wrapped and raw values just fine).